api/grant.go

```diff
@@ -35,13 +35,13 @@ func (r *CreateGrantResponse) StatusCode() int {
 }
 
 type ListGrantsRequest struct {
-	User          uid.ID `form:"user"`
-	Group         uid.ID `form:"group"`
-	Resource      string `form:"resource" example:"production.namespace"`
-	Destination   string `form:"destination" example:"production"`
-	Privilege     string `form:"privilege" example:"view"`
-	ShowInherited bool   `form:"showInherited" note:"if true, this field includes grants that the user inherits through groups"`
-	ShowSystem    bool   `form:"showSystem" note:"if true, this shows the connector and other internal grants"`
+	User          uid.ID `form:"user" note:"ID of user granted access" example:"6TjWTAgYYu"`
+	Group         uid.ID `form:"group" note:"ID of group granted access" example:"6k3Eqcqu6B"`
+	Resource      string `form:"resource" example:"production.namespace" note:"a resource name in Infra's Universal Resource Notation"`
```

Comment:
```suggestion
	Resource      string `form:"resource" example:"production.namespace" note:"a resource name"`
```

I don't think we should mention a Universal Resource Notation as a first-class concept since we don't document it, but perhaps having a good example with a namespace could show users what this is

Reply:
Yup, I think that was there before.